test/auth/features/oidc/broker.go

```diff
@@ -55,28 +65,27 @@ func BrokerSendEventWithOIDCTokenToSubscriber() *feature.Feature {
 	f.Setup("install broker", broker.Install(brokerName, broker.WithEnvConfig()...))
 	f.Setup("broker is ready", broker.IsReady(brokerName))
 	f.Setup("broker is addressable", broker.IsAddressable(brokerName))
+	f.Setup("Broker has HTTPS address", broker.ValidateAddress(brokerName, addressable.AssertHTTPSAddress))
 
 	// Install the sink
 	f.Setup("install sink", eventshub.Install(
 		sink,
+		eventshub.StartReceiverTLS,
 		eventshub.OIDCReceiverAudience(sinkAudience),
-		eventshub.StartReceiver))
-
-	// Install the trigger and Point the Trigger subscriber to the sink svc.
-	f.Setup("install trigger", trigger.Install(
-		triggerName,
-		brokerName,
-		trigger.WithSubscriberFromDestination(&duckv1.Destination{
-			Ref:      service.AsKReference(sink),
-			Audience: &sinkAudience,
-		}),
 	))
+
+	f.Setup("install the trigger and specify the CA cert of the destination", func(ctx context.Context, t feature.T) {
```

Comment:
I would simply name this step `install the trigger`, as you do some other configuration besides the CA too. Same in the other files.